### PR TITLE
fix water temp display units

### PR DESCRIPTION
**Pull request by Rashmil-1999**
This PR will fix the units in stations environment table.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [unreleased]
+
+### Fixed
+
+- Error in displaying temperatures at depths in station details.
```
Comment (navarroc) [inline CHANGELOG.md:12]: Can you make a github issue for this so we can start recording issues in the changelog?

**Comment by navarroc** [comment]
Deployed on dev

**Review by navarroc — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [unreleased]
+
+### Fixed
+
+- Error in displaying temperatures at depths in station details. [#96](https://github.com/Oceans-1876/challenger-portal/issues/97)
```
Comment (navarroc) [inline CHANGELOG.md:12]: Should be #97, you have #96 which matches the PR not the issue

**Review by navarroc — CHANGES_REQUESTED** [review]
Can you please change #96 to #97 to match the issue?

File: CHANGELOG.md (hunk 2 above)
Comment (Rashmil-1999) [reply CHANGELOG.md:12]: ohh Shit :( I didn't see this comment. On it!

**Review by navarroc — APPROVED** [review]

**Pull request merged**